File: Java/Soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class Soma_vetor {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int n;
		double soma, media;

		System.out.print("Quantos numeros voce vai digitar? ");
		n = sc.nextInt();

		double[] vet = new double[n];

		for (int i = 0; i < n; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
		}

		System.out.println();
		System.out.print("VALORES = ");
		for (int i = 0; i < n; i++) {
			System.out.print(String.format("%.1f", vet[i]) + " ");

		}
		System.out.println();

		soma = 0;
		for (int i = 0; i < n; i++) {
			soma = soma + vet[i];
		}
		System.out.println("SOMA = " + String.format("%.2f", soma));

		media = soma / n;
		System.out.println("MEDIA = " + String.format("%.2f", media));

		sc.close();
	}
}

File: C/soma_vetor.c
#include <stdio.h>

int main()
{
    int n;
    double soma, media;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    double vet[n];

    for (int i = 0; i < n; i++)
    {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }
    printf("\nVALORES = ");

    for (int i = 0; i < n; i++)
    {
        printf("%.1lf ", vet[i]);
    }
    printf("\n");

    soma = 0;
    for (int i = 0; i < n; i++)
    {
        soma = soma + vet[i];
    }
    printf("SOMA = %.2lf\n", soma);

    media = soma / n;
    printf("MEDIA = %.2lf", media);

    return 0;
}
